core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/TableRepairJob.java

```diff
@@ -61,16 +61,24 @@
         myRepairLockType = builder.repairLockType;
     }
 
+    @Override
     public TableReference getTableReference()
     {
         return myTableReference;
     }
 
+    @Override
     public RepairConfiguration getRepairConfiguration()
     {
         return myRepairConfiguration;
     }
 
+    @Override
+    public RepairStateSnapshot getRepairStateSnapshot()
```

Comment:
Is there any test cases for this new method? (or any of the methods in the RepairJobView interface)?

Reply:
Added a test case to check all arguments of RepairJobView.